### ISSUE TITLE
Print CSS in Firefox + Safari

### ISSUE THREAD

**Issue by ssssoz** [sparcopen/infrastructure#76]
- [ ] Print on MacOS Safari: `<header>` has its own cover page; the rest of the text is in a narrow column which doesn’t look too awful
- [ ] Print on MacO Firefox: a lot of white space around `<header>`; blank page

- [MacOS-Chrome.pdf](https://github.com/sparcopen/infrastructure/files/4798940/MacOS-Chrome.pdf)
- [MacOS-Safari.pdf](https://github.com/sparcopen/infrastructure/files/4798942/MacOS-Safari.pdf)
- [MacOS-Firefox.pdf](https://github.com/sparcopen/infrastructure/files/4798941/MacOS-Firefox.pdf)

**Comment by JosephMcArthur** [comment]
I think this is likely best solved by using the chrome view to make a downloadable version, and then providing that. What do you think @sssoz? Lazy... but it'll work.

**Comment by ssssoz** [comment]
@JosephMcArthur Do you mean we manually generate each post and upload it as the corresponding PDF for each post? Seems like more work.

**Comment by JosephMcArthur** [comment]
Yeah, I guess it is but it seemed like doing this was a pain for you so I didn't want you to have to worry given the limited time (and, likely, pay off given the % of people using them).

Joseph McArthur
Assistant Director: Right to Research Coalition ( http://righttoresearch.org )
Director / Founder: Open Access Button ( http://openaccessbutton.org )
Twitter: @Mcarthur_Joe ( https://twitter.com/Mcarthur_Joe )
Skype: joseph_mcarthur

**Comment by JosephMcArthur** [comment]
A random thought. If we're unhappy with how these get printed and think it'll be a pain, it may be best just to hid the buttons on safari / firefox so that folks get a good UX... even if it's not a complete one.
